CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to `otp_builds`
+
+1. [License](#license)
+1. [Reporting a bug](#reporting-a-bug)
+1. [Requesting or implementing a feature](#requesting-or-implementing-a-feature)
+1. [Submitting your changes](#submitting-your-changes)
+   1. [Code Style](#code-style)
+   1. [Committing your changes](#committing-your-changes)
+   1. [Pull requests and branching](#pull-requests-and-branching)
+
+## License
+
+`otp_builds` is licensed under [Apache License Version 2.0](LICENSE.md) License, for all code.
+
+## Reporting a bug
+
+`otp_builds` is not perfect software and will be buggy.
+
+Bugs can be reported via
+[GitHub issues: bug report](https://github.com/erlef/otp_builds/issues/new?template=bug_report.md).
+
+Some contributors and maintainers may be unpaid developers working on `otp_builds`, in
+their own time, with limited resources. We ask for respect and understanding, and will provide the
+same back.
+
+## Requesting or implementing a feature
+
+Before requesting or implementing a new feature, do the following:
+
+- search, in existing [issues](https://github.com/erlef/otp_builds/issues) (open or closed), whether
+the feature might already be in the works, or has already been rejected,
+- make sure you're using the latest software release (or even the latest code, if you're going for
```

Comment:
latest software release?

Reply:
I'm not sure how the repository will progress, but it's surely possible to imagine multiple version of the images, and if an image failed locally before opening a pull request / bug report the poster should figure out if the image they're using has not been recreated.